notebooks/hidden_features/HiddenFeatures.ipynb

```diff
@@ -26,40 +26,22 @@
     "from certifai.scanner.builder import (CertifaiScanBuilder, CertifaiPredictorWrapper, CertifaiModel, CertifaiModelMetric,\n",
     "                                      CertifaiDataset, CertifaiGroupingFeature, CertifaiDatasetSource,\n",
     "                                      CertifaiPredictionTask, CertifaiTaskOutcomes, CertifaiOutcomeValue)\n",
-    "from certifai.scanner.report_utils import scores, construct_scores_dataframe"
+    "from certifai.scanner.report_utils import scores, construct_scores_dataframe\n",
+    "from certifai.common.utils.encoding import CatEncoder"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 2,
    "metadata": {},
-   "outputs": [],
-   "source": [
-    "# special import - \n",
-    "# for multiprocessing to work in a Notebook,  pickled classes must be in a separate package or notebook\n",
-    "# hence, the encoder class has to be somewhere other than the current notebook\n",
-    "# from ipynb.fs.defs.cat_encoder import CatEncoder # <- doesn't work on Azure Notebooks\n",
-    "# %run cat_encoder.py # <- doesn't work because code doesn't remain external\n",
-    "\n",
-    "# Azure Notebooks workaround - \n",
-    "import os\n",
-    "import sys\n",
-    "sys.path.append(\"../utils\")\n",
-    "from cat_encoder import CatEncoder"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 3,
-   "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
       "Accuracy of modal predictor: 0.7\n",
-      "Model 'aware' accuracy is 0.82\n",
-      "Model 'unaware' accuracy is 0.818\n"
+      "Model 'aware' accuracy is 0.781\n",
+      "Model 'unaware' accuracy is 0.867\n"
```

Comment:
This makes no sense.  No way should the unaware model have significantly higher accuracy!

Reply:
Added test/train split, which makes everything better

Reply:
typo "modal" -> model